main.nf

```diff
@@ -19,12 +19,9 @@ params.fasta                = WorkflowMain.getGenomeAttribute(params, 'fasta')
 params.fai                  = WorkflowMain.getGenomeAttribute(params, 'fai')
 params.vep_cache            = WorkflowMain.getGenomeAttribute(params, 'vep_cache')
 params.bwa                  = WorkflowMain.getGenomeAttribute(params, 'bwa')
-params.gnomad_sv            = WorkflowMain.getGenomeAttribute(params, 'gnomad_sv')
```

Comment:
These are now set values and no longer optional?

Reply:
Aren't gnomad annotations part of the annotsv annotations? Otherwise we can add a config for vcfanno for gnomad_sv

Reply:
From the AnnotSv docs it seems the Sv set for both GRCh37 and 38 is available in AnnotSV yes. Good call.